top.wake

```diff
@@ -24,7 +24,15 @@ global def compileSimTarget dut buildRoot =
 global topic vivadoVsrcHooks : (dut: DUT) => Option (Unit => List Path)
 global topic vivadoBlackBoxResources : DUT => Option (List String)
 
+global def fpgaSynthTarget board dut buildRoot =
+  setUpBitstream board dut buildRoot
+  | fpgaSynthOnly
+
 global def bitstreamTarget board dut buildRoot =
+  setUpBitstream board dut buildRoot
+  | makeBitstream
+
+global def setUpBitstream board dut buildRoot =
```

Comment:
```suggestion
def setUpBitstream board dut buildRoot =
```

Can you make this a private function so that no one outside of this file can use it? It minimizes our API surface area.

Reply:
Done, though I am not 100% sure that nothing outside this file uses this function as is